checkov/common/checks_infra/solvers/attribute_solvers/not_within_attribute_solver.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional, Any, Dict
+
+from checkov.common.checks_infra.solvers.attribute_solvers.within_attribute_solver import WithinAttributeSolver
+from checkov.common.graph.checks_infra.enums import Operators
+
+
+class NotWithinAttributeSolver(WithinAttributeSolver):
```

Comment:
it actually would have been better to add the new operator in a separate PR, because you also need to add tests for it, like here #3482 doing so would also add it to the changelog and now it will be a hidden feature addition.

Reply:
Completely agree so I removed this solver, instead I used a `not` operator on the `within` one.
I would have just dropped the commit but I made a mess and pushed something else with it, so I just added another commit removing it :/